src/modules/dotnet.rs

```diff
@@ -196,6 +207,7 @@ fn get_dotnet_file_type(path: &Path) -> Option<FileType> {
     match extension_lower.as_ref().map(|f| f.as_ref()) {
         Some("sln") => return Some(FileType::SolutionFile),
         Some("csproj") | Some("fsproj") | Some("xproj") => return Some(FileType::ProjectFile),
+        Some("props") | Some("targets") => return Some(FileType::MsBuildFile),
```

Comment:
the general classification of these files are msbuild specific files, i didn't see a reason to break it out right now